project_national/housing_characteristics/Refrigerator.tsv

```diff
@@ -1,24 +1,11 @@
-Single-Family Attached	High	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Average	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	Medium	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Low	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	High	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Average	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Medium	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	Low	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	High	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Average	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Medium	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	Low	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
-Mobile Home	High	0.024380	0.066827	0.321826	0.562587	0.024380	133.250000	1710517.000000
-Multi-Family with 2 - 4 Units	Average	0.013683	0.073747	0.322583	0.576304	0.013683	230.250000	2243989.500000
-Multi-Family with 5+ Units	Medium	0.014750	0.061490	0.300636	0.607870	0.014727	475.000000	4721506.000000
-Single-Family Attached	Low	0.015515	0.066510	0.281868	0.620327	0.015502	221.750000	1674581.625000
-Single-Family Detached	High	0.012364	0.069471	0.256486	0.648904	0.012347	1935.750000	17833094.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
+Mobile Home	0.054273	0.008690	0.035322	0.195062	0.617901	0.088753	0.000000	533.000000	6842068.000000
+Multi-Family with 2 - 4 Units	0.030439	0.005122	0.041124	0.212894	0.642879	0.067543	0.000000	921.000000	8975958.000000
+Multi-Family with 5+ Units	0.032757	0.005397	0.033313	0.179132	0.681388	0.065999	0.002014	1900.000000	18886024.000000
+Single-Family Attached	0.035360	0.005510	0.035532	0.187268	0.623666	0.111545	0.001119	887.000000	6698326.500000
+Single-Family Detached	0.028144	0.004337	0.037080	0.180561	0.606475	0.141354	0.002049	7743.000000	71332376.000000
 # Created by: sources\recs\2009\tsv_maker.py
 # Source: Constructed using U.S. EIA 2009 Residential Energy Consumption Survey (RECS) microdata.
-# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficiencies by year.
+# Source: Age of refrigerator converted to efficiency levels using ENERGYSTAR shipment-weighted efficienciesby year data from Home Energy Score: http://hes-documentation.lbl.gov/. Check the comments in: HES-Refrigerator_Age_vs_Efficiency.tsv
```

Comment:
White space `efficienciesby` --> `efficiencies by`. Needs to be fixed in resstock-estimation.